web/src/rendered_markdown.ts

```diff
@@ -167,24 +171,31 @@ export const update_elements = ($content: JQuery): void => {
                 if (person === undefined) {
                     people.add_inaccessible_user(user_id);
                 }
-                wrap_mention_content_in_dom_element(this);
+                wrap_mention_content_in_dom_element(this, false);
```

Comment:
We can use function default to `false` to avoid changing this call everwhere to pass the `false` value.